client/xmlrpc_client_test.go

```diff
@@ -66,3 +71,53 @@ func TestExecuteCall(t *testing.T) {
 		})
 	}
 }
+
+func TestExecuteCallWithTimeout(t *testing.T) {
+	tt := []struct {
+		name           string
+		connectTimeout int
+		requestTimeout int
+		sleepTime      time.Duration
```

Comment:
perhaps we can rename this filed to something more meaningful, like "serverResponseTime" or similar